Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Widgets/AssetEditorToolbar/AssetEditorToolbar.cpp

```diff
@@ -179,11 +179,17 @@ namespace GraphCanvas
 
             SceneRequestBus::EventResult(selectedElements, m_activeGraphId, &SceneRequests::GetSelectedNodes);
 
-            if (selectedElements.size() == 1)
+            for (const auto& selectedElement : selectedElements)
```

Comment:
Does this loop need anything before it to capture ungrouping multiple groups as a single undoable operation?

Reply:
I could do that, might help with undo/redo, I'll try it

Reply:
I tried it, but it did not change the undo/redo behavior and it only adds a touch of complexity, I'll keep the change as is, it's easier to read